Description: Add missing verses of Cohen's song
Author: Andrea Colangelo <warp10@ubuntu.com>
Bug-Debian: http://bugs.debian.org/719017
Last-Update: 2013-08-24

--- fortune-mod-1.99.1.orig/datfiles/songs-poems
+++ fortune-mod-1.99.1/datfiles/songs-poems
@@ -1308,6 +1308,16 @@ And everybody knows it's now or never.
 And everybody knows that you live forever when you've done a line or two.
 Everybody knows the deal is rotten: Old Black Joe's still pickin' cotton
 for you ribbons and bows.  And everybody knows.
+
+And everybody knows that the Plague is coming. Everybody knows that it's
+moving fast. Everybody knows that the naked man and woman. Are just a shining
+artifact of the past. Everybody knows the scene is dead. But there's gonna be
+a meter on your bed. That will disclose. What everybody knows.
+
+And everybody knows that you're in trouble. Everybody knows what you've been
+through. From the bloody cross on top of Calvary. To the beach of Malibu.
+Everybody knows it's coming apart. Take one last look at this Sacred Heart.
+Before it blows. And everybody knows.
 		-- Leonard Cohen, "Everybody Knows"
 %
 Everything's great in this good old world;
