Description: Add quote to datfiles/computers
Author: Andrea Colangelo <warp10@ubuntu.com>
Last-Update: 2013-09-06

--- fortune-mod-1.99.1.orig/datfiles/computers
+++ fortune-mod-1.99.1/datfiles/computers
@@ -5555,3 +5555,8 @@ Step 1: Close AutoCAD - I know this can
 also wait for AutoCAD to crash if you want.
     -- Michael Rotolo
        (http://themadcadder.blogs.com/my_weblog/2008/12/external-xref-manager-xref-path-saver.html)
+%
+When I was young I always daydreamed about being a programmer/hacker in a
+dystopian cyberpunk future. I don't know whether or not I should be alarmed that
+my daydreams and reality are converging.
+  -- ghc (http://hn.meteor.com/posts/6244882-01d32)
